### PR TITLE
Add custom metric - observability

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/6428

### CODE REVIEW

File: observability/manage_observe.adoc [hunk 1]
```diff
@@ -54,6 +53,48 @@ data:
 
 A custom rule is created.
 
+[#adding-custom-metrics]
+=== Adding custom metrics
+
+Add metrics to the `metrics_list.yaml` file, in order to view collected metrics that are from managed clusters. *Note*: When you update your custom rules, `observability-observatorium-thanos-rule` pods are restarted automatically.
```
Comment (clyang82) [inline observability/manage_observe.adoc:58]: Why you mention update your custom rules? This is for customizing metrics. not for customizing the rules.

**Review by dislbenn — APPROVED** [review]

**Comment by dockerymick** [comment]
@dislbenn Thank you for the review!

File: observability/manage_observe.adoc [hunk 2]
```diff
@@ -54,6 +52,48 @@ data:
...
+
+. Log in to your cluster.
+. Verify that `mco observability` is enabled. Check for the following message in the `status.conditions.message` reads: `Observability components are deployed and running`. Run the following command:
++
+----
+oc get mco observability -o yaml
+----
+
+. Access the `observability-metrics-custom-whitelist.yaml` file in the 
+`open-cluster-management-observability` namespace by running the following command:
```
Comment (clyang82) [inline observability/manage_observe.adoc:70]: by default, there is no `observability-metrics-custom-whitelist` configmap in observability namespace. the user needs to create this configmap, for example:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: observability-metrics-custom-whitelist
  namespace: open-cluster-management-observability
data:
  metrics_list.yaml: |
    names:
      - node_memory_Active_bytes
```

**Comment by clyang82** [comment]
@dockerymick sorry for late response. I just found that I forgot the click "submit" my reviews so that the comments are in Pending status.

**Comment by dockerymick** [comment]
@clyang82 no worries, thank you for your comments, I will apply the suggestion in a different PR

File: observability/manage_observe.adoc (hunk 1 above)
Comment (dockerymick) [reply observability/manage_observe.adoc:58]: @clyang82 Should Adding custom metrics have a lower Header? H3 to H2. In the file, I would remove an equal sign (=) on line 57

**Pull request merged**